src/app/Components/Details/Note/index.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import moment from 'moment';
+import './index.scss';
```

Comment:
nitpick: Personally, I look for whole file imports at the _bottom_ of other dependencies. I wouldn't expect to see them half way through.

I know we don't have a linting rule for dependency order though - so totally up to you to make a call on if you want to update or not :)

Reply:
Good point! All fixed up now